projects/developer/src/app/navbar/navbar.component.html

```diff
@@ -27,7 +30,7 @@
             </div>
             <p-slideMenu #menu [model]="itemsMobile" [popup]="true" [viewportHeight]="340"></p-slideMenu>
             <button pButton type="button" class="navbar__mobile-nav-button" icon="fa fa-bars"
-                    (click)="menu.toggle($event)"></button>
+                (click)="menu.toggle($event)"></button>

```

Comment:
Although these are good changes, fixing linting/styling clutters up PRs. I would prefer to see separate PRs for just those. A PR should have a clear description of what is being changed and the diff should clearly match the description.